src/hb-ot-math.h

```diff
@@ -204,6 +204,20 @@ typedef enum {
   HB_OT_MATH_KERN_BOTTOM_LEFT = 3
 } hb_ot_math_kern_t;
 
+/**
+ * hb_ot_math_kern_entry_t:
+ * @max_correction_height: The maximum height at which this entry should be used
+ * @kern_value: The kern value of the entry
+ *
+ * Data type to hold math kerning (cut-in) information for a glyph.
+ *
+ * Since: 3.4.0
```

Comment:
Use, `REPLACEME` for the version number, we will update it when cutting the release with the actual release number.

Reply:
Thanks, now fixed.